framework/base/ErrorHandler.php

```diff
@@ -262,7 +275,12 @@ public function handleError($code, $message, $file, $line)
      */
     public function handleFatalError()
     {
-        unset($this->_memoryReserve);
+        $this->_memoryReserve = null;
+
+        // fix working directory for some Web servers
+        if (!in_array(PHP_SAPI, ['cli', 'phpdbg'], true)) {
```

Comment:
When/why does it matter?

What SAPIs are changing working dir during shutdown?

Reply:
@samdark see PR description

Reply:
https://www.php.net/manual/en/function.register-shutdown-function.php